spec/declaration.dd

```diff
@@ -55,6 +55,12 @@ $(GNAME DeclaratorIdentifierList):
 $(GNAME DeclaratorIdentifier):$(LEGACY_LNAME2 VarDeclaratorIdentifier)
     $(GLINK_LEX Identifier)
     $(GLINK_LEX Identifier) $(GLINK2 template, TemplateParameters)$(OPT) $(D =) $(GLINK Initializer)
+    $(GLINK BitfieldDeclarator)
+
+$(GNAME BitfieldDeclarator):
+    $(D :) $(GLINK2 expression, AssignExpression)
+    $(GLINK_LEX Identifier) $(D :) $(GLINK2 expression, AssignExpression)
```

Comment:
```suggestion
    $(GLINK_LEX Identifier)$(OPT) $(D :) $(GLINK2 expression, AssignExpression)
```

Reply:
Note the above trivial change will conflict with the important grammar change below so maybe ignore it.

Reply:
They're both equivalent.